partner_contact_birthdate/models/res_partner.py

```diff
@@ -11,3 +14,12 @@ class ResPartner(models.Model):
     _inherit = "res.partner"
 
     birthdate_date = fields.Date("Birthdate")
+    age = fields.Integer(string="Age", readonly=True, compute="_compute_age")
+
+    @api.depends("birthdate_date")
+    def _compute_age(self):
+        for record in self:
+            age = 0
+            if record.birthdate_date:
+                age = relativedelta(fields.Date.today(), record.birthdate_date,).years
```

Comment:
Why the comma after birthdate_date?

Reply:
@NL66278 : I have remove comma after birthdate_date